docs/supported-sources/github.md

```diff
@@ -0,0 +1,49 @@
+# GitHub
+
+[GitHub](https://github.com/) is a developer platform that allows developers to create, store, manage and share their code.
+
+ingestr supports GitHub as a source.
+
+## URI format
+
+The URI format for GitHub is as follows:
+
+```plaintext
+github://?access_token=<access_token>&owner=<owner>&repo=<repo>
+```
+
+URI parameters:
+
+- `access_token`: Access Token used for authentication with the GitHub API
```

Comment:
does GitHub require access token for public repositories? if not, then I suggest stating that here.

also, if token is not required for public please add an integration test in `main_test.py` file for this endpoint.